flowapi/tests/unit/test_geography.py

```diff
@@ -48,6 +51,7 @@
     )
 
 
+@pytest.mark.xfail(reason="This test may need to be reworked...")
```

Comment:
Why?

Reply:
Hmm, I took a look at this again because I was wondering whether I had marked it `xfail` during refactoring. But after having a second look I think it's either not needed any more, or we need to adjust what exactly it tries to test. A third option is that the logic in the `get_geography()` route needs tweaking.